File: app/src/main/java/com/example/vkwall/data/model/ProfileWall/PostSource.java
package com.example.vkwall.data.model.ProfileWall;

import com.google.gson.annotations.SerializedName;

public class PostSource {
    @SerializedName("type")
    private String type;
    @SerializedName("platform")
    private String platform;
    @SerializedName("data")
    private String data;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getPlatform() {
        return platform;
    }

    public void setPlatform(String platform) {
        this.platform = platform;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }
}

File: app/src/main/java/com/example/vkwall/data/model/Recommended/PostSource.kt
package com.example.vkwall.data.model.Recommended

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PostSource {
    @SerializedName("type")
    @Expose
    var type: String? = null
    @SerializedName("platform")
    @Expose
    var platform: String? = null
}
